Makefile

```diff
@@ -5,7 +5,7 @@ IMAGE=ansible-tower-container
 TAG=latest
 
 image/build:
-	@(cd image; docker build -t ${REG}/${ORG}/${IMAGE}:${TAG} .)
+	@(cd image/tower_base; docker build -t ${REG}/${ORG}/${IMAGE}:${TAG} .)
```

Comment:
Might be worth having two makefiles, one for each image. We'd also then need to update the README to reflect these changes.